app/src/test/java/com/duckduckgo/app/survey/ui/SurveyViewModelTest.kt

```diff
@@ -64,22 +68,31 @@ class SurveyViewModelTest {
 
     private var mockAppBuildConfig: AppBuildConfig = mock()
 
+    private val mockAppDaysUsedRepository: AppDaysUsedRepository = mock()
+
+    private val mockSurveyRepository: SurveyRepository = mock()
+
     private lateinit var testee: SurveyViewModel
+    private val testSource = SurveySource.IN_APP
 
     @Before
     fun before() {
         MockitoAnnotations.openMocks(this)
-
-        whenever(mockAppBuildConfig.versionName).thenReturn("name")
-
-        testee = SurveyViewModel(
-            mockSurveyDao,
-            mockStatisticsStore,
-            mockAppInstallStore,
-            mockAppBuildConfig,
-            coroutineTestRule.testDispatcherProvider,
-        )
-        testee.command.observeForever(mockCommandObserver)
+        runTest {
```

Comment:
Why is everything under `runTest`? I guess this is only because of `getLastActiveDay` and probably we should use `runBlocking` instead